src/tiled/automapper.cpp

```diff
@@ -124,6 +139,17 @@ bool AutoMapper::setupRuleMapProperties()
                        "Ignoring this property.")
                 .arg(mRulePath, name, value.toString()) + QLatin1Char('\n');
     }
+
+    // OverflowBorder and WrapBorder make no sense for infinite maps
+    if (mMapWork->infinite()) {
+        mOptions.overflowBorder = false;
+        mOptions.wrapBorder = false;
+    }
+
+    // Each of the border options imply MatchOutsideMap
+    if (mOptions.overflowBorder || mOptions.wrapBorder)
+        mOptions.matchOutsideMap = true;
+
```

Comment:
As requested, I put the options to make the border options imply **MatchOutsideMap** and prevent infinite maps from turnin on the border options.